src/lib/app.js

```diff
@@ -55,6 +56,16 @@ class App {
     koaApp.ws.use(passport.initialize())
     koaApp.ws.use(websocketRouter.routes())
     koaApp.ws.use(websocketRouter.allowedMethods())
+    // Override listen() in order to specify a maxPayload. See:
+    //   https://github.com/kudos/koa-websocket/blob/785001c20d94fbb60cb05596bb1869889f539351/index.js#L15-L20
+    // for the original version.
+    koaApp.ws.listen = function (server) {
+      this.server = new WebSocketServer({
+        server: server,
+        maxPayload: 64 * 1024
```

Comment:
It would be good to put this as a constant up at the top of the file and provide a justification for the size chosen